src/utils/rest-api.js

```diff
@@ -456,10 +456,13 @@ export function duplicateCase(
 }
 
 export function elementExists(directoryUuid, elementName, type) {
+    const elementNameEncoded = new URLSearchParams({
+        elementName: elementName,
```

Comment:
you can simply use this shortcut using destructuring
`new URLSearchParams({ elementName }).toString()`